scrapy_zyte_api/providers.py

```diff
@@ -66,6 +151,7 @@ async def __call__(  # noqa: C901
         http_response = None
         screenshot_requested = Screenshot in to_provide
         for cls in list(to_provide):
+            self._track_auto_fields(crawler, request, cast(type, cls))
```

Comment:
Maybe you can add a setting to disable this in case it becomes an issue for some people?